### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Discuss proposals
+    url: https://groups.google.com/g/elixir-ecto
+    about: Discuss proposals
+  - name: Ask Questions
+    url: https://elixirforum.com/
+    about: Ask and answer questions here.
+
```
Comment (josevalim) [inline .github/ISSUE_TEMPLATE/config.yml:10]:
```suggestion
blank_issues_enabled: false

contact_links:
  - name: Discuss proposals
    url: https://groups.google.com/g/elixir-ecto
    about: Send proposals for new ideas in the mailing list.
  - name: Ask Questions
    url: https://elixirforum.com/
    about: Ask and answer questions on ElixirForum.

```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  - type: textarea
+    id: current-behavior
+    attributes:
+      label: Current behavior
+      description: How can we reproduce what you're seeing? Include code samples, errors and stacktraces if appropriate.
+      placeholder: |-
+        1. foo
+        2. bar
+        3. baz
```
[the remark this answers is not in the document]
Comment (yordis) [reply .github/ISSUE_TEMPLATE/bug.yml:46]: Chrome in MacOS ....

Screen Shot 2022-06-01 at 11 44 59 AM

[the remark this answers is not in the document]
Comment (warmwaffles) [reply .github/ISSUE_TEMPLATE/bug.yml:46]: Firefox in linux

**Pull request merged**